api/.nycrc.json

```
@@ -0,0 +1,10 @@
+{
+  "all": true,
+  "reporter": "html",
+  "include": [
+    "lib/**/*.js"
+  ],
+  "exclude": [
+    "eslintrc.js"
+  ]
```

Je pense qu'il serait préférable de mettre les répertoires générés ailleurs que dans le répertoire courant :

* pour ne pas encombrer le répertoire `./pix/api/` qui est déjà bien rempli
* `/tmp` est un bon candidat car :
   * ces données de couverture de tests sont par nature temporaire
   * `/tmp` est souvent (au moins sur système Linux) un répertoire en RAM, ce qui à la fois accélère potentiellement un peu le traitement et à la fois économise la durée de vie du SSD sur lequel se trouve le répertoire `./pix/api/`

```suggestion
  ],
  "temp-dir": "/tmp/test-coverage-temp-pix-api",
  "report-dir": "/tmp/test-coverage-pix-api"
```

---

Excellente idée, ça évite de mettre à jour le `.gitignore` et autres détails.
J'ai mis `/tmp`,si un Windows l'utilise on utilisera [tmpdir()](https://www.geeksforgeeks.org/node-js-os-tmpdir-method/)

---

Corrigé